cmd/zdb/zdb.c

```diff
@@ -3415,6 +3418,20 @@ match_object_type(dmu_object_type_t obj_type, uint64_t flags)
 	return (match);
 }
 
+static void
+dump_spill_bp(dnode_t *dn)
+{
+	zbookmark_phys_t zb;
+
+	SET_BOOKMARK(&zb, dmu_objset_id(dn->dn_objset), dn->dn_object, 0,
+	    DMU_SPILL_BLKID);
+
+	printf("\nSpill block: ");
+	print_indirect(dmu_objset_spa(dn->dn_objset),
+	    DN_SPILL_BLKPTR(dn->dn_phys), &zb, dn->dn_phys);
```

Comment:
Using `print_indirect()` feels a bit misleading since a spill block is just a directly attached block.  It's even exactly a real L0 block.  What do you think about just using `snprintf_blkptr_compact()` here?

Reply:
Sounds good, will do.